server/src/pages/shot/shotpage-header.js

```diff
@@ -107,7 +108,7 @@ exports.ShotPageHeader = class ShotPageHeader extends React.Component {
               hasFxa={this.props.isFxaAuthenticated}>
         { myShotsText }
         { shotInfo }
-        <div className="shot-alt-actions">
+        <div className={classnames({"shot-alt-actions": this.props.isOwner})}>
```

Comment:
For !owned shots, we want download button to show on the same row as shot title and don't need shot-alt-actions class that renders alt action buttons in separate row on medium and small window widths